### PR TITLE
custom server requirements.txt file generation added to the notebook

### LINKED ISSUE #710: missing mentioning the requirements.txt in the "serving custom model page"
Although I can see it is in the [directory](https://github.com/SeldonIO/MLServer/blob/master/docs/examples/custom/requirements.txt) it is not mentioned in the [webpage](https://mlserver.readthedocs.io/en/latest/examples/custom/README.html) and will result in error for people copy/pasting from the webpage and not using the notebook.

### PR DESCRIPTION
**Pull request by saeid93**
Fixes #710

### CODE REVIEW

File: docs/examples/custom/README.ipynb [hunk 1]
```diff
@@ -335,6 +335,26 @@
...
+    "MLServer will automatically find your requirements.txt file and install necessary python packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile models.py\n",
```
Comment (adriangonz) [inline docs/examples/custom/README.ipynb:352]: Should this line be `%%writefile requirements.txt` instead?

**Review by adriangonz — CHANGES_REQUESTED** [review]
Nice one @saeid93! Thanks a lot for adding this one.

Just added a small comment below about the file name. Once that's addressed, I think this should be good to go! :+1:

File: docs/examples/custom/README.ipynb (hunk 1 above)
Comment (saeid93) [reply docs/examples/custom/README.ipynb]: Sure, That's why I shouldn't do all the pull requests late at night. :D
Should work now. Please check.

**Review by adriangonz — APPROVED** [review]
Nice one! Thanks for adding the changes.

Once the tests are green, this should be good to go :+1:

**Pull request merged**